config/Development.toml

```diff
@@ -18,6 +18,14 @@ port = 5432
 dbname = "hyperswitch_db"
 pool_size = 5
 
+[replica_database]
+username = "rep"
+password = "rep"
+host = "localhost"
+port = 5432
+dbname = "hyperswitch_db"
+pool_size = 5
```

Comment:
Not exactly a blocker, but could you update the username and password to be slightly more descriptive, `replica_user` and `replica_pass` maybe? And the database name, could you make it `replica_db` instead?

If you're making this change, please make this change in `config.example.toml` as well.

Reply:
I have changed the username and pasword, as we are using the same db but different user, so I have kept hyperswitch_db only.